javers-core/src/main/java/org/javers/core/JaversCore.java

```diff
@@ -187,6 +188,12 @@ public Diff initial(Object newDomainObject) {
         return (List)queryRunner.queryForShadows(query);
     }
 
+    @Override
+    public <T> List<Shadow<T>> findShadows(List<JqlQuery> queries) {
```

Comment:
List of queries to perform one query is not ok, please add multiple ID support to  proper Filter

Reply:
I removed it. But one test from core don't pass.

Reply:
what now?